ms-campus-kid-grades/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.5'
```

Comment:
This file is not necessary for the project architecture, we already have a `docker-compose` file.